product/roundhouse.console/Program.cs

```diff
@@ -429,6 +432,12 @@ private static void run_diff_utility(ConfigurationPropertyHolder configuration)
             }
         }
 
+        public static void init_security_protocol()
+        {
+            // allow tls
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls | SecurityProtocolType.Tls11 | SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;

```

Comment:
Does this work in .NET 3.5? .NET 4.0? I thought this was only available once you moved up to .NET 4.5+?